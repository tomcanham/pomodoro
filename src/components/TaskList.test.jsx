import React from 'react'
import { AddTodo, Todo, TaskList } from './TaskList'
import renderer from 'react-test-renderer'

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <AddTodo addTodo={(() => 1)} value="foo" setValue={(() => 2)} />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()

  // re-rendering
  tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
