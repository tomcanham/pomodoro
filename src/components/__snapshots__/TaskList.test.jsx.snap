// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<form
  onSubmit={[Function]}
>
  <h3>
    Add Todo:
  </h3>
  <input
    name="text"
    onChange={[Function]}
    type="text"
    value="foo"
  />
</form>
`;

exports[`Link changes the class when hovered 2`] = `
<form
  onSubmit={[Function]}
>
  <h3>
    Add Todo:
  </h3>
  <input
    name="text"
    onChange={[Function]}
    type="text"
    value="foo"
  />
</form>
`;
